README.md

```diff
@@ -48,13 +48,8 @@ I'm not far from completing [milestone 0.1.0](https://github.com/buzz-language/b
 cd mir
 make
 ```
-4. Add to your shell rc:
-```bash
-export BUZZ_PATH="/path/to/buzz/dist"
-export PATH="$BUZZ_PATH/bin:$PATH"
-```
-5. Build it: `zig build -Doptimize=ReleaseSafe -p dist`
```

Comment:
I think we should still advice `-Doptimize=ReleaseSafe` to prevent people from running a debug build which is significantly slower

Reply:
Maybe there's a way to select it by default in `build.zig`?

Reply:
there is, but the advice might be better